scrapy/extensions/feedexport.py

```diff
@@ -306,6 +353,17 @@ def _exporter_supported(self, format):
             return True
         logger.error("Unknown feed format: %(format)s", {'format': format})
 
+    def _batch_deliveries_supported(self, uri):
+        """
+        If FEED_STORAGE_BATCH_SIZE setting is specified uri has to contain %(time)s or %(batch_id)s
+        to distinguish different files of partial output
+        """
+        if not self.storage_batch_size:
```

Comment:
Could you combine these three conditions into one single `ìf` check? Also, please include an explicit `return False`. It's not strictly needed because there is an implicit `None` being returned, which evaluates to `False`, but [_Explicit is better than implicit_](https://www.python.org/dev/peps/pep-0020).

Also, should the log level be "warning" instead? :thinking:

Reply:
Done.
I thought it can be more readable if I slit it into 2 `if`.